<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Darwin Agudelo</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FLUKE®️ VT650</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>515</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QComboBox" name="cbserial">
     <property name="geometry">
      <rect>
       <x>75</x>
       <y>8</y>
       <width>160</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="lserial">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>70</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Serial Port:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bconfig">
     <property name="geometry">
      <rect>
       <x>245</x>
       <y>8</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Config</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bstart">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>335</x>
       <y>8</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bstop">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>425</x>
       <y>8</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Stop</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QFrame" name="frame_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>515</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QWidget" name="layoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>8</y>
       <width>501</width>
       <height>27</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="horizontalSpacing">
       <number>5</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="lpressure">
        <property name="text">
         <string>Pressure:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lepression">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string notr="true"/>
        </property>
        <property name="frame">
         <bool>true</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="lflow">
        <property name="text">
         <string>Flow:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="leflow">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="4">
       <widget class="QLabel" name="lvolume">
        <property name="text">
         <string>Volume:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <widget class="QLineEdit" name="levol">
        <property name="dragEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="gvdata">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>780</width>
      <height>590</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>710</y>
      <width>780</width>
      <height>40</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <widget class="QLabel" name="lsfile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>File path:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lefilename">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>8</y>
       <width>600</width>
       <height>24</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string notr="true"/>
     </property>
     <property name="frame">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="bsgrap">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>8</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Save graph</string>
     </property>
    </widget>
   </widget>
   <widget class="QGraphicsView" name="gvlogo">
    <property name="geometry">
     <rect>
      <x>535</x>
      <y>10</y>
      <width>255</width>
      <height>90</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>781</width>
      <height>591</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2"/>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfig">
    <property name="title">
     <string>Config</string>
    </property>
    <widget class="QMenu" name="menuSample_rate">
     <property name="title">
      <string>Sample rate</string>
     </property>
     <addaction name="action25_Hz"/>
     <addaction name="action50_Hz"/>
     <addaction name="action100_Hz"/>
     <addaction name="action150_Hz"/>
     <addaction name="action200_Hz"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="menuSample_rate"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuConfig"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action25_Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>25 Hz</string>
   </property>
  </action>
  <action name="action50_Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50 Hz</string>
   </property>
  </action>
  <action name="action150_Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>150 Hz</string>
   </property>
  </action>
  <action name="action200_Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>200 Hz</string>
   </property>
  </action>
  <action name="action100_Hz">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100 Hz</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>cbserial</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
